e2e_samples/config_driven_data_pipelines/README.md

```diff
@@ -1,106 +1,188 @@
 # Config Driven Data Pipelines
 
-**Build:**
+The sample demonstrate how an end to end data pipeline can be deployed by changing the configuration of an existing template implemented in Azure Databricks.
 
-## Getting Started
+## Contents <!-- omit in toc -->
 
-These instructions will allow you to build the Webapi component on your local machine for development and testing purposes.
+- [Solution Overview](#solution-overview)
+  - [Archictecture](#architecture)
+  - [Folder structure](#folder-structure)
+- [How to use the sample](#how-to-use-the-sample)
+  - [Prerequisites](#prerequisites)
 
-This code is built using the [Cake](https://cakebuild.net/) build framework and [Maven](https://maven.apache.org/) as packet manager. It can be built using locally installed components or a containerized environment.
+  - [Run the example](#setup-and-deployment)
 
-The recommended editor for working on this code is [Visual Studio Code](https://code.visualstudio.com/Download). The recommended extensions can be viewed by running the VSCode command "Extensions: Show Recommended Extensions". Once the Cake extension is installed, run the VSCode command "Cake: Install intellisense support" to make it easier to work on the build scripts.
 
-VSCode commands can be executed on _command pallete_ which can be launched using Ctrl+Shift+P key combination.
+  - [Known Issues, Limitations and Workarounds](#known-issues-limitations-and-workarounds)
+ 
+- [Key Learnings](#key-learnings)
+  
 
-### Containerized Enviornments
 
 
+---------------------
+## Solution Overview
+The diagram below showed the overall architecture of the solution. 
 
-### Building on Windows
+<img src= "./docs/images/architecture.png" width="800px"/>

```

Comment:
seems like the all the images links are breaking

Reply:
It looks all good from my side. This is the snapshot from the preview version of the PR.  We can have a quick call on the document view from your side. 
![image](https://user-images.githubusercontent.com/37413937/182311330-c26a9548-c982-4db4-a1ca-aa3f0dbecf78.png)